backend/data/processors/courses_processing.py

```diff
@@ -157,6 +158,17 @@ def process_enrolment_rules(processed: dict, course: dict):
     processed["raw_requirements"] = re.sub(
         "<br/><br/>", "", course["enrolment_rules"])
 
+def process_multi_term(processed: dict, formatted: dict) -> None:
+    is_multiterm = False
+
+    for attribute in formatted["attributes"]:
+        if attribute["type"] == "multi-term_course":
+            is_multiterm = True
+
+    if int(formatted["UOC"]) % 6 == 0:
```

Comment:
is this `% 6` required here? Are there any courses that are `6 uoc` and multi-term? If so, do we want to exclude them?

Not sure if `0 uoc` courses are in the scope of this card but, if there's multi-term courses that cost `0 uoc`, wouldn't this make their `is_multiterm` false which (i dont think) is what we want

Reply:
@imagine-hussain the only 0uoc courses are the cringe commerce courses so its ok

Reply:
> is this % 6 required here? Are there any courses that are 6 uoc and multi-term? If so, do we want to exclude them?

UNSW has 2 types of multi-term courses. 

There are courses like `ENGG2600` or `ENGG3600` where you have to do the same course multiple times to get credit for it. There are other courses like `ACCT3202 (Industry placement 2)` and `ACCT3203 (Industry placement 3)` which are  a multi-group course, in which you need to complete several different courses to get credit for all of them).

The handbook doesn't actually differentiate between them. The multi-term_course attribute simply says
```
This course is taught across multiple terms or is a member of a group of courses taken across multiple terms. In either case a final grade is withheld until the final course is completed.
```

From looking at the courses, it seems that the UOC of all 'multi-group courses' is divisible by 6 (e.g. `ACCT3202` and `ACCT3203` both have 6 UOC), whereas multi-term courses all have UOCs that aren't divisible by 6 (e.g. 2, 3, 10, etc).

I've excluded multi-group courses from being counted as 'multiterm' because the logic I've applied to multiterm courses like `ENGG2600` isn't applicable for them.

>  if there's multi-term courses that cost 0 uoc, wouldn't this make their is_multiterm false
I'm not fully up to date with 0 UOC courses, but they don't seem to be in coursesFormattedRaw.json, which is what the parser works on. If there were some multiterm courses that had 0 UOC, then parts of my logic would need to be re-written because my algorithm would assume that they need to be repeated infinite times.

Reply:
Checks out